src/StreamService.Broker/StreamService.Broker.csproj

```diff
@@ -0,0 +1,15 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <TargetFramework>net6.0</TargetFramework>
+    <ImplicitUsings>enable</ImplicitUsings>
+    <Nullable>enable</Nullable>
+  </PropertyGroup>
+
+  <Import Project="$(ProjectDir)..\..\Common.Configurations.targets" />

```

Comment:
project gets target framework from Common.Configuration.targets here (line 9), no need in line 4

You can move these properties to Common.Configuration.targets too and remove them from other .csproj files
```
<ImplicitUsings>enable</ImplicitUsings>
<Nullable>enable</Nullable>
```

Reply:
by the way, what do these lines mean?
```
<ImplicitUsings>enable</ImplicitUsings>
<Nullable>enable</Nullable>
```

Reply:
it was generated by vs and later fixed